### PR TITLE
A small StFcsDb update

### PR DESCRIPTION
**Pull request by akioogawa**
Adding a function to get STAR coordinate XYZ for a StFcsCluster.

### CODE REVIEW

**Review by starsdong — APPROVED** [review]
Looks straightforward

**Comment by starsdong** [comment]
Daniel, could you take a quick review of this update? I think it still needs your approval as the code owner before merging.

File: StRoot/StFcsDbMaker/StFcsDb.h
Comment (plexoos) [inline StRoot/StFcsDbMaker/StFcsDb.h]: I'd define the parameter as const

```suggestion
  StThreeVectorD getStarXYZ(const StFcsCluster* clu, float FcsZ=-1.0) const;           //from StFcsCluster
```

Comment (akioogawa) [reply StRoot/StFcsDbMaker/StFcsDb.h]: Done. For both with StFcsClutser and StFcsHit.

**Review by plexoos — APPROVED** [review]

**Comment by starsdong** [comment]
@jdbrice Daniel, any further comments before we merge?

**Comment by starsdong** [comment]
Hi Dmitri, could you please merge this PR? Thanks

**Pull request merged**
